src/plugin/modules/widgets/reads_set.js

```diff
@@ -0,0 +1,228 @@
+/*global define */
+/*jslint white: true, browser: true, plusplus: true */
+define([
+    'kb/common/html',
+    'kb_sdk_clients/SetAPI/dev/SetAPIClient',
+    'kb/service/client/workspace',
+    '../utils',
+    'text!../../resources/html/reads_set.html',
+    'bluebird',
+    'jquery',
+    'underscore',
+    'handlebars',
+    'numeral',
+    'datatables',
+    'datatables_bootstrap'
+    ],
+    function (html, SetAPI, Workspace, utils, RSTemplate, Promise,
+              $, _, handlebars, Numeral, DataTable, DataTable_BS) {
```

Comment:
Generally if a module is only loaded for side effect, as in jQuery plugins and styles, you might consider leaving the receiving argument off, since it implies they will be used. Leaving them for the end of the module list is thus good practice, as is done here, so that the arguments may be omitted.